Compile error on latest v5.6.4.2

No problem in ca797e114b0640d6aa1a573ea68526a9263df81e, the following error when using 60222a568da1954bff35526733ed2cecfbfc96d3:

```
In file included from /var/lib/dkms/8812au/5.6.4.2_35491.20191025/build/include/drv_types.h:37,
                 from /var/lib/dkms/8812au/5.6.4.2_35491.20191025/build/core/rtw_io.c:47:
/var/lib/dkms/8812au/5.6.4.2_35491.20191025/build/include/drv_types_linux.h: In function ‘eth_hw_addr_set’:
/var/lib/dkms/8812au/5.6.4.2_35491.20191025/build/include/drv_types_linux.h:22:42: error: expected ‘;’ before ‘}’ token
   22 |     memcpy(dev->dev_addr, addr, ETH_ALEN)
      |                                          ^
```

Probably here missing a semicolon? 
https://github.com/aircrack-ng/rtl8812au/blob/09ba98c7d3a5bdea4a7df5c385efc46bc28b467e/include/drv_types_linux.h#L22

--- a/include/drv_types_linux.h
+++ b/include/drv_types_linux.h
@@ -19,7 +19,7 @@
 /* Porting from linux kernel v5.15 48eab831ae8b9f7002a533fa4235eed63ea1f1a3 3f6cffb8604b537e3d7ea040d7f4368689638eaf*/
 static inline void eth_hw_addr_set(struct net_device *dev, const u8 *addr)
 {
-    memcpy(dev->dev_addr, addr, ETH_ALEN)
+    memcpy(dev->dev_addr, addr, ETH_ALEN);
 }
 #endif
 

